libchannelserver/ChannelServer.cpp

```diff
@@ -36,11 +36,22 @@ void dev::channel::ChannelServer::run()
 {
     auto sslCtx = m_sslContext->native_handle();
     auto cert = SSL_CTX_get0_certificate(sslCtx);
+    // get cert failed
+    if (!cert)
+    {
+        CHANNEL_LOG(ERROR) << LOG_DESC(
+            "run ChannelServer failed for get cert from the sslContext failed");
+        BOOST_THROW_EXCEPTION(
+            dev::channel::ChannelException("init ChannelServer failed for get certificat failed"));
+    }
     /// get issuer name
     const char* issuer = X509_NAME_oneline(X509_get_issuer_name(cert), NULL, 0);
-    std::string issuerName(issuer);
-    m_certIssuerName = issuerName;
-    OPENSSL_free((void*)issuer);
+    if (issuer)
```

Comment:
If can't get issuer should throw an exception to exit or log a warning to off sdk issuer check